### ISSUE TITLE
AJP support has been deprecated in nginx 1.9

### ISSUE THREAD

**Issue by chuegel** [Keyfactor/ejbca-community-helm#5]
As of this commit https://github.com/kubernetes/ingress-nginx/pull/10158 the AJP support has been deprecated.
Please consider changing the helm chart to reflect this change.

**Comment by rriemann** [comment]
I have spent much time to figure out why the AJP example in the readme is not working. I am glad I found the answer now. Please reopen this issue until the readme is updated.

**Comment by svenska-primekey** [comment]
Good call. I will open this. Thanks for the comment to get this back on my radar.
